sw/device/lib/testing/test_rom_ext/test_rom_ext_irq_vector.S

```diff
@@ -4,7 +4,7 @@
 
 /**
  * @file
```

Comment:
This was done to stay consistent with the file names used with the test ROM, e.g. [`test_rom_irq_vector.S`](https://github.com/lowRISC/opentitan/blob/master/sw/device/lib/testing/test_rom/test_rom_irq_vector.S). However now that you bring this to my attention, I think it might be even better to stay consistent with the production ROM/ROM_EXT and configure the IRQ vector in the `asm` startup code (see [here](https://github.com/lowRISC/opentitan/blob/47387221a4d3d8236d1efe767a96bc62cafb818b/sw/device/silicon_creator/rom_ext/rom_ext_start.S#L40)), rather than in a separate file.

I have filed a separate issue about this (#9787), and will address this in a separate PR if that's alright with you? (since it involves changes to the `test_rom` too).

Reply:
SGTM, Should have said that was just a nit.

Reply:
Hehe no worries, good catch!